<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddModDialog</class>
 <widget class="QDialog" name="AddModDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Install Mod</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="5" column="0" colspan="3">
    <widget class="QScrollArea" name="options_container">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>595</width>
        <height>118</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="fomod_label">
       <property name="toolTip">
        <string>This deployer's target directory is used by the fomod installer to check for file dependencies</string>
       </property>
       <property name="text">
        <string>Fomod target:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="fomod_deployer_box"/>
     </item>
    </layout>
   </item>
   <item row="8" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="toolTip">
      <string>Options for the installer</string>
     </property>
     <property name="text">
      <string>Options:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>The new mod will be added to these installers</string>
     </property>
     <property name="text">
      <string>Add to deployers:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QListWidget" name="deployer_list">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Mod name</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ValidatingLineEdit" name="name_text">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>Mod version</string>
       </property>
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ValidatingLineEdit" name="version_text">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QTreeWidget" name="content_tree">
     <property name="toolTip">
      <string>Mod contents</string>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="toolTip">
        <string>Removes the first n directories from every path</string>
       </property>
       <property name="text">
        <string>Root level:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="root_level_box">
       <property name="suffix">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="group_check">
       <property name="toolTip">
        <string>Only one mod in a group can be active at a time</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="ValidatingLineEdit" name="group_field">
       <property name="placeholderText">
        <string>enter mod name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="group_combo_box">
       <item>
        <property name="text">
         <string>Add to Group</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Replace Mod</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QComboBox" name="installer_box"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>Determines how mods are installed</string>
       </property>
       <property name="text">
        <string>Installer:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/validatinglineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddModDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddModDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
